widget/entry.go

```diff
@@ -237,12 +237,15 @@ func (e *Entry) DragEnd() {
 //
 // Implements: fyne.Draggable
 func (e *Entry) Dragged(d *fyne.DragEvent) {
+	pevt := d.PointEvent
+	// substract scroll offset because we need visible column and row numbers to render
```

Comment:
I'm not sure that a comment is needed as the code is pretty clear.
If you do think it should remain perhaps it should be about correcting coordinates rather than rendering selection, as that is not what this code is doing.

Reply:
At least for me, looking at the code it is not clear what is happening. A better comment should be this?
"subtract scroll offset because we need visible column and row numbers to correct coordinates, not the total ones."

Reply:
Isn't that the opposite of what is happening?
The event is relative to the top-left of the widget (Entry) so to know how far into the scroll content we should go it is required to adjust by `Offset` amount (offset will be negative, as the content is left of the widget it is in).

Reply:
> The event is relative to the top-left of the widget (Entry) so to know how far into the scroll content we should go it is required to adjust by Offset amount

Yes, but in the case of selection you need column number relative to what is visible.

> (offset will be negative, as the content is left of the widget it is in)

Hmm I will do some tests. Although I thought at the most-left part of the widget, the offset is 0, but I will investigate.

Reply:
> The event is relative to the top-left of the widget (Entry) so to know how far into the scroll content we should go it is required to adjust by Offset amount (offset will be negative, as the content is left of the widget it is in).

You are right, the confusing part is maybe that `getRowCol` function (called in `updateMousePointer`) makes the rows and columns relative to the whole widget instead of the top-left visible, to mitigate that effect I subtract the scroll offset that is added inside `getRowCol`.

Reply:
Comment updated. Done.